fastlane/Fastfile

```diff
@@ -176,7 +178,7 @@ platform :android do
 end
 
 def increment_version_in(previous_version, new_version, path)
-  sed_regex = 's/' + previous_version + '/' + new_version + '/'
+  sed_regex = 's/' + previous_version + '/' + new_version + '/g'
```

Comment:
we should always have been doing `/g` otherwise it only replaces the first instance in the file